java/daikon/LogHelper.java

```diff
@@ -22,6 +22,7 @@ private LogHelper() {
 
   // Class variables so user doesn't have to use "Level." prefix.
   public static final Level FINE = Level.FINE;
+  public static final Level FINER = Level.FINER;
```

Comment:
Rather than defining all these variables, it would be possible to use `static import` to permit them to be used without a prefix in this file.  Within other files, I don't see the benefit of using `LogHelper.FINER` vs. `Level.FINER`.

Reply:
done